.github/workflows/test.yml

```diff
@@ -25,11 +25,12 @@ jobs:
           openai_api_key: ${{ secrets.OPENAI_API_KEY }}
           openai_base_url: ${{ secrets.OPENAI_BASE_URL }}
           github_token: ${{ secrets.GH_TOKEN }}
-          model: 'gpt-4-turbo-preview'
+          model: 'gpt-4o-mini'
```

Comment:
🔄 Updated model from 'gpt-4-turbo-preview' to 'gpt-4o-mini'. Ensure this change aligns with your intended functionality.